### DISCUSSION TITLE
rust-notify support

### DISCUSSION THREAD

**Discussion by thePaulRichard** [starship/starship#2731] [Q&A] [answered]
Hello, how can I install starship with rust-notify support?

**Comment by chipbuster** [comment, marked as answer]
You will likely need to install from source, since (as far as I know) no official builds or binaries exist with notify-rust support.

You'll need to [install rustup + cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html), then run the following command:

```
cargo install starship --features=notify-rust
```

This will download and build the appropriate code for starship with rust-notify (might take a while). If there is a new version available, you can install it with `cargo install starship --force --features=notify-rust` (unfortunately cargo does not have support for intelligent updates at the moment).

**Reply by thePaulRichard** [reply]
Thank you @chipbuster !
